pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -802,134 +802,57 @@ var _ = Describe("Transform", func() {
 			})
 		})
 
-		When("project service label instructing to expose the k8s service is specified as `domain.name`", func() {
-			ingressAnnotations := map[string]string{
-				"kubernetes.io/ingress.class":    "external",
-				"cert-manager.io/cluster-issuer": "prod-le-dns01",
-			}
+		When("project service extension instructing to expose the k8s service is specified as `domain.name`", func() {
 			domain := "domain.name"
 			path := "path"
 
 			BeforeEach(func() {
 				projectService.SvcK8sConfig.Service.Expose.Domain = filepath.Join(domain, path)
-				projectService.SvcK8sConfig.Service.Expose.IngressAnnotations = ingressAnnotations
-
-				// projectService.Labels = composego.Labels{
-				// 	config.LabelServiceExpose: filepath.Join(domain, path),
-				// }
 			})
 
-			It("initialises Ingress with a port routing to the project service name and specifies host information", func() {
-				ing := k.initIngress(projectService, port)
+			It("specifies host in the initialised Ingress", func() {
+				ingress := k.initIngress(projectService, port)
+				Expect(ingress.Spec.Rules[0].Host).To(Equal(domain))
+			})
 
-				Expect(ing).To(Equal(&networkingv1beta1.Ingress{
```

Comment:
I'm not sure if there is another spec covering the entire ingress object like the one removed here but if not I think I'd be good to add it.